app/Http/Controllers/ClxMessagesController.php

```diff
@@ -160,7 +161,7 @@ public function transmit(RclMessage $rclMessage, ClxMessageRequest $request)
             'entry_time_restriction' => $entryRequirement ?? null,
             'raw_entry_time_restriction' => $request->get('entry_time_requirement'),
             'free_text' => $isReclearance ? '** RECLEARANCE '.now()->format('Hi').' ** '.$request->get('free_text') : $request->get('free_text'),
-            'datalink_authority' => DatalinkAuthorities::from($request->get('datalink_authority')),
+            'datalink_authority_id' => DatalinkAuthority::whereId($request->get('datalink_authority'))->first()->id,
```

Comment:
this seems roundabout and unnecessary? the ID is right there!

Reply:
https://github.com/vatsimnetwork/nattrak/pull/160/commits/a662997a44373e51b4d8d90d1f67da610efcdcd2